src/vs/workbench/api/node/extHostTerminalService.ts

```diff
@@ -597,6 +593,12 @@ export class ExtHostTerminalService implements ExtHostTerminalServiceShape {
 				throw new Error('Not implemented');
 			}
 		} as IWorkspaceFolder : null;
+
+		// Get the initial cwd
+		const terminalConfig = configProvider.getConfiguration('terminal.integrated');
+
+		const initialCwd = terminalEnvironment.getCwd(shellLaunchConfig, os.homedir(), lastActiveWorkspace ? lastActiveWorkspace : undefined, this._variableResolver, activeWorkspaceRootUri, terminalConfig.cwd);
```

Comment:
> lastActiveWorkspace ? lastActiveWorkspace : undefined

Is the ternary needed?

Reply:
lastActiveWorkspace can be null in this case but the function doesn't take null it takes undefined so I need to convert null to undefined which is what the ternary is used for

Reply:
K, we can leave it as is for now then